### DISCUSSION TITLE
新三编译出来咋没有无线

### DISCUSSION THREAD

**Discussion by xiaiohuan** [immortalwrt/immortalwrt#811] [Q&A]
新路由3 编译出来 默认跑一一遍出来 没有做任何更改 没有无线 后台管理界面无线选项也没有

**Comment by 1715173329** [comment]
> https://httpd.apache.org/docs/2.4/getting-started.html#:~:text=Troubleshooting%20any%20problem%20without%20the,you%20how%20to%20fix%20it.

Troubleshooting any problem without the error log is like driving with your eyes closed.
